VideoLocker/src/main/java/org/edx/mobile/model/course/CourseStructureJsonHandler.java

```diff
@@ -45,15 +45,16 @@ public CourseStructureV1Model processInput(String jsonInput)
     class BlockDataDeserializer implements JsonDeserializer<BlockData>
     {
         @Override
-        public BlockData deserialize(JsonElement json, Type typeOfT, JsonDeserializationContext context) throws JsonParseException
-        {
+        public BlockData deserialize(JsonElement json, Type typeOfT,
+                                     JsonDeserializationContext context) throws JsonParseException {
```

Comment:
I guess all the deserializer classes here could use a bit of line-wrapping and auto-formatting. Also note that these classes should all be static.